classes/Domain/Services/ResetEmailer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace OpenCFP\Domain\Services;
+
+class ResetEmailer
```

Comment:
IOW, this would be useful if we're required to switch out Swift for something else, but would we ever?